CONTRIBUTING.md

```diff
@@ -35,6 +35,13 @@ We require new unit tests to be contributed with any new functionality added.
 
 Before sending pull requests, make sure your changes pass formatting, linting and unit tests. These checks will run with the pull request builds. Alternatively, you can run the checks manually on your local machine [as specified below](#development).
 
+#### Dependencies
+If additional new Python module dependencies are required, think about where to put them:
+
+- If they're required for fms-hf-tuning, then append them to the end of the requirements.txt file.
```

Comment:
We want to deprecate the requirements.txt file. Let's use pyproject.toml to manage dependencies.

Reply:
Thank you Ted, I will adjust ASAP once we decide to switch entirely to the toml file.  Really the only use-case I see for keeping the requirements.txt file is if a developer wanted all the pre-reqs without installing the fms-hf-tuning module yet.  If that's not real use-case, then the toml file is a lot cleaner way to manage the dependencies.